Graphics/GraphicsEngine/interface/PipelineState.h

```diff
@@ -75,21 +75,28 @@ typedef struct SampleDesc SampleDesc;
 struct ShaderResourceVariableDesc
 {
     /// Shader stages this resources variable applies to. More than one shader stage can be specified.
-    SHADER_TYPE                   ShaderStages DEFAULT_INITIALIZER(SHADER_TYPE_UNKNOWN);
+    SHADER_TYPE                    ShaderStages DEFAULT_INITIALIZER(SHADER_TYPE_UNKNOWN);
 
     /// Shader variable name
-    const Char*                   Name         DEFAULT_INITIALIZER(nullptr);
+    const Char*                    Name         DEFAULT_INITIALIZER(nullptr);
 
     /// Shader variable type. See Diligent::SHADER_RESOURCE_VARIABLE_TYPE for a list of allowed types
-    SHADER_RESOURCE_VARIABLE_TYPE Type         DEFAULT_INITIALIZER(SHADER_RESOURCE_VARIABLE_TYPE_STATIC);
+    SHADER_RESOURCE_VARIABLE_TYPE  Type         DEFAULT_INITIALIZER(SHADER_RESOURCE_VARIABLE_TYPE_STATIC);
+    
+    /// AZ TODO
+    SHADER_RESOURCE_VARIABLE_FLAGS Flags        DEFAULT_INITIALIZER(SHADER_RESOURCE_VARIABLE_FLAG_DEFAULT);
```

Comment:
We decided we will not need it. The variables will be automatically merged for one shader type (e.g. there will be the same `CameraAttribs` buffer in all RAY_HIT shaders, but RAY_GEN shader may bind another resouce to `CameraAttribs` variable.
We will implement this functionality later.